src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+@import "../utils/_mixin.scss";
+@import "../utils/_placeholders.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid $color-border-card;
+  border-radius: 5px;
+  width: $width-card;
+  padding: 32px 16px 16px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__container {
+    width: $container-cards;
+  }
+
+  &__image {
+    @include container(160px);
+    display: block;
+    height: 134px;
+  }
+
+  &__caption {
+    margin: 40px 0 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin: 4px 0 0;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $color-code;
+  }
+
+  &__rating {
+    @extend %flexSpaceBetween;
+    margin: 16px 0 0;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    margin: 0;
+  }
+
+  &__price {
+    @extend %flexSpaceBetween;
+    margin: 24px 0 0;
+    line-height: 18px;
+
+    &-text {
+      margin: 0;
+      color: $color-code;
+    }
+
+    &-amount {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: block;
+    margin-top: 16px;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    background-color: $color-teal;
+    font-size: 14px;
+    line-height: 40px;
+    font-weight: 700;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+
+    &:hover {
+      border: 1px solid $color-teal;
```

Comment:
Don't add border on hover. Add it for default state of element, and only change its color on hover